webapp/src/Todo/AddTodo.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+
+const styles = {
```

Comment:
Why dont you put styles in css?

Reply:
im thinking about to put all styles into js files. 
Found this more comfortable to use with bigger projects. And keep css only for common cases..